AI/tests/evaluate_model.py

```
@@ -200,22 +200,19 @@ def evaluate_market_signals():
+            # 결과 누적
             global_y_true.extend(y_test)
             global_y_pred.extend(y_pred_class)
             global_returns.extend(r_test)
             
             processed_count += 1
             
         except Exception as e:
-            # 데이터 불량 등으로 인한 개별 종목 에러는 무시하고 진행
-            print(f"[Skip] {ticker}: {e}")
-            pass
+            pass # 에러 난 종목은 건너뜀
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)